public/app/features/backup/components/StorageLocations/StorageLocationsActions/StorageLocationsActions.test.tsx

```diff
@@ -2,11 +2,31 @@ import React from 'react';
 import { shallow } from 'enzyme';
 import { DBIcon } from '../../DBIcon';
 import { StorageLocationsActions } from './StorageLocationsActions';
-import { StorageLocation } from '../StorageLocations.types';
+import { LocationType, StorageLocation } from '../StorageLocations.types';
 
 describe('StorageLocationsActions', () => {
   it('should have DBIcon', () => {
-    const wrapper = shallow(<StorageLocationsActions location={(null as unknown) as StorageLocation} />);
+    const wrapper = shallow(
+      <StorageLocationsActions onUpdate={jest.fn()} location={(null as unknown) as StorageLocation} />
+    );
     expect(wrapper.find(DBIcon).exists()).toBeTruthy();
   });
+
+  it('should call onUpdate', () => {
+    const location: StorageLocation = {
+      locationID: 'Location1',
+      name: 'location_1',
+      description: 'first location',
+      type: LocationType.S3,
+      path: 's3://foo',
+    };
+    const onUpdate = jest.fn();
```

Comment:
Nit: I know this is a test, but generally speaking handler functions should be named differently from the event listener attribute, starting with `handle`, not `on`. E.g.: `<a onClick={handleClick}>...</a>`

Reply:
done